lib/matplotlib/figure.py

```diff
@@ -32,7 +32,7 @@
 
 from matplotlib.cbook import Stack, iterable
 
-from matplotlib import _image
+from matplotlib import image as mimage
```

Comment:
I sometimes have thoughts that `__all__ = []` should be added to all python files :smiling_imp: